mk/sm.spec.in

```diff
@@ -36,19 +38,20 @@ rm -rf $RPM_BUILD_ROOT
 [ ! -x /sbin/chkconfig ] || chkconfig --add sm-multipath
 service sm-multipath start
 %systemd_post make-dummy-sr.service
+%systemd_post mpcount.service
 %systemd_post snapwatchd.service
-%systemd_post updatempppathd.service
-%systemd_post usb-scan.service
+%systemd_post xs-sm.service
```

Comment:
Unrelated changed?

Reply:
sort of, synchronising the spec with the one being used for product

Reply:
As the spec needed changing in order to remove the files and services anyway.